packages/stable/src/api/sessions/types.ts

```diff
@@ -0,0 +1,107 @@
+import type { EpochTimestamp, FilterQuery } from '../../types';
+
+export type SessionStatus =
+  | 'READY'
+  | 'ACTIVE'
+  | 'CANCELLED'
+  | 'EXPIRED'
+  | 'REVOKED'
+  | 'ACCESS_LOST';
+
+export const SessionStatus = {
+  READY: 'READY' as SessionStatus,
+  ACTIVE: 'ACTIVE' as SessionStatus,
+  CANCELLED: 'CANCELLED' as SessionStatus,
+  EXPIRED: 'EXPIRED' as SessionStatus,
+  REVOKED: 'REVOKED' as SessionStatus,
+  ACCESS_LOST: 'ACCESS_LOST' as SessionStatus,
+};
```

Comment:
```suggestion
export const SessionStatus: Record<string, SessionStatus> = {
  READY: 'READY',
  ACTIVE: 'ACTIVE',
  CANCELLED: 'CANCELLED',
  EXPIRED: 'EXPIRED',
  REVOKED: 'REVOKED',
  ACCESS_LOST: 'ACCESS_LOST',
};
```

Reply:
`union type SessionStatus + const SessionStatus` is quite a strong way to go for instead of the classic enums, as it gives best of both worlds.

Reply:
we could ofc use, `Record<SessionStatus, SessionStatus>` but it still not quite the same...

Reply:
we could ofc use, `Record<SessionStatus, SessionStatus>` but it still not quite the same...